spring-integration-graphql/src/main/java/org/springframework/integration/graphql/outbound/GraphQlMessageHandler.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2021-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.graphql.outbound;
+
+import java.util.Collections;
+import java.util.Locale;
+import java.util.Map;
+
+import javax.annotation.Nullable;
+
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.expression.Expression;
+import org.springframework.expression.common.LiteralExpression;
+import org.springframework.expression.spel.support.StandardEvaluationContext;
+import org.springframework.graphql.GraphQlService;
+import org.springframework.graphql.RequestInput;
+import org.springframework.integration.expression.ExpressionUtils;
+import org.springframework.integration.expression.SupplierExpression;
+import org.springframework.integration.handler.AbstractReplyProducingMessageHandler;
+import org.springframework.messaging.Message;
+import org.springframework.util.Assert;
+
+/**
+ * A {@link org.springframework.messaging.MessageHandler} capable of fielding GraphQL Query, Mutation and Subscription requests.
+ *
+ * @author Daniel Frey
+ * @since 6.0
+ */
+public class GraphQlMessageHandler extends AbstractReplyProducingMessageHandler {
+
+	private final GraphQlService graphQlService;
+
+	private StandardEvaluationContext evaluationContext;
+
+	private Expression queryExpression;
+
+	private Expression operationNameExpression = new SupplierExpression<>(() -> null);
+
+	private Expression variablesExpression = new SupplierExpression<>(() -> Collections.emptyMap());
+
+	private Locale locale = null;
```

Comment:
Must `@Nullable` as well.
No need in the `= null`. Because it is `null` by default anyway.